src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1150,8 +1150,7 @@ function tryRemoveElement(elem) {
                 return true;
             }
 
-            // This is unfortunate, but we're unable to remove the textTrack that gets added via addTextTrack
```

Comment:
Removed because this comment is out of date. We have implementations working [here](https://github.com/jellyfin/jellyfin-web/blob/e95b457567fd7e6d8d2ef62287f31b3b8e971b4a/src/plugins/htmlVideoPlayer/plugin.js#L1254) and [here](https://github.com/jellyfin/jellyfin-web/blob/e95b457567fd7e6d8d2ef62287f31b3b8e971b4a/src/plugins/htmlVideoPlayer/plugin.js#L1263)